### ISSUE TITLE
ALE for categorical features

### ISSUE THREAD

**Issue by jklaise** [SeldonIO/alibi#233]
ALE currently supports numerical features only. An extension to categorical features is possible, but comes with serious caveats for interpretability (see https://compstat-lmu.github.io/iml_methods_limitations/ale-misc.html) so I think some more research needs to be done before investing in categorical feature support.

**Comment by jklaise** [comment]
Tagged as low priority as it is unclear how to proceed with categorical features. An alternative would be to implement `PDP` which does support categorical features natively.
